GitVersion.yml

```diff
@@ -1,5 +1,4 @@
 assembly-versioning-scheme: Major
-next-version: 1.0
```

Comment:
What is the effect of removing this @bording ?

Reply:
The only impact this will have is for brand new repos that don't have a release branch at all yet. That means that for the initial work, `master` would default to 0.x prereleases, but as soon as a `release-1.0` branch is created, you'd get a 1.0 package.

And if you want master to be 1.0 even before the first release branch, then you can increment with the commit message stuff.

And if you're talking about a repo that already has a `release-1.0` branch (and we've released the 1.0.0 package) then absolutely nothing changes, because `master will be creating 1.1 prerelease packages at that point.

Reply:
This behavior will be handy for Platform Previews, which will always be released as a 0.x version.